alerts/KubeNodeNotReady.md

```diff
@@ -0,0 +1,42 @@
+# KubeNodeNotReady
+
+## Meaning
+
+This alert is fired when a Kubernetes node is not in `Ready` state for a certain period. In this case, the node is not able to host any new pods.
+
+## Impact
+
+Depending on the overall workload and the type of the node, this can affect the performance of the cluster deployements.
+
+## Diagnosis
+
+The notification details should list the node that's not ready. For Example:
+
+```txt
+ - alertname = KubeNodeNotReady
+...
+ - node = achvte-rplds-w-b-c5l96.us-east4-b.c.o-58140aae.internal
```

Comment:
Please use `node1.example.com` or some such dummy hostname for any nodes. Runbooks should be as generic as possible.